06_29_2018/reverse_integer.py

```diff
@@ -0,0 +1,16 @@
+#! usr/bin/python
+#Write a program to reverse a 3digit integer
+
+
+
+#input = raw_input("Enter a 3 digit integer:")
+#above gives a string. reverse operation can only be done on a string
+
+#function below reverses the integer
+def reverse_integer(input):
```

Comment:
input is name of the keyword in python. You should use a different variable name.